log/logging.go

```diff
@@ -135,19 +134,14 @@ func SetPrefix(prefix string) {
 // Fatal outputs a severe error message just before terminating the process.
 // Use judiciously.
 func Fatal(id, description string, keysAndValues ...interface{}) {
-	if Level < LevelFatal {
-		return
-	}
-	DefaultLogger.logMessage(LevelFatalName, id, description, keysAndValues...)
-	osExit(1)
+	keysAndValues = append([]interface{}{"id", id}, keysAndValues...)
```

Comment:
My only concern is that this is a public repo and we should be more careful with it than we might be internally.

I do like what the change is accomplishing, however. @bdotdub Thoughts?